### PR TITLE
Bundle ID name will be hidden if it is too long, tooltip will appear to display full name

### PR DESCRIPTION
**Pull request by MaksymShokin**
@tonytlwu @squallstar 

## Issue
Fliplet/fliplet-studio#4668

## Description
Limited max length on the bundle ID input to 155 chars. Added bootstrap tooltip. Bundle ID name will be hidden if it is too long.

## Screenshots/screencasts
bundleid demo

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: css/interface.css [hunk 1]
```diff
@@ -107,6 +107,19 @@
   margin: 0;
 }
 
+.bundleId-ast-text,
```
Comment (squallstar) [inline css/interface.css]: `ast`?

- appStore: `appStore`
- ent: `enterprise`
- uns: `unsigned`

File: js/interface.js [hunk 2]
```diff
@@ -691,6 +691,18 @@ function checkGroupErrors() {
...
+
+$('[data-toggle="tooltip"]').tooltip({
+  title: function() {
+    var tooltipText = $(this).text();
+    if (tooltipText.length < 41) {
+      return;
+    }
+    return tooltipText;
+  },
+  delay: { "show": 500, "hide": 300 }
```
Comment (squallstar) [inline js/interface.js]: double quotes are unnecessary here

File: css/interface.css (hunk 1 above)
Comment (MaksymShokin) [reply css/interface.css]: @squallstar It was called like this before. Should I change the name?

File: js/interface.js (hunk 2 above)
Comment (MaksymShokin) [reply js/interface.js]: @squallstar That how bootstrap does it. Change to single quotes?

File: css/interface.css (hunk 1 above)
Comment (squallstar) [reply css/interface.css]: yes please, not sure what `ast` meant but if it's related to Google Play / App Store then we should use `store` for consistency.

File: js/interface.js (hunk 2 above)
Comment (squallstar) [reply js/interface.js]: @MaksymShokin you don't need quotes in Javascript object keys unless they have a dash or other non-common characters:

```js
{ foo: 'bar' }
```

```js
{ 'foo-bar': 'baz' }
```

Our code guidelines clearly state that:
1. Quotes should only be added when necessary
2. You should use single quote whenever possible

Comment (MaksymShokin) [reply js/interface.js]: @squallstar Fixed both.

**Pull request merged**